app/views/templates/_form.html.erb

```diff
@@ -37,8 +37,9 @@
 
 
       <% if pxe_with_building_hosts?(@template) -%>
+        <% warning_text = (_("Warning: The template is associated to at least one host in build mode. To apply the change, disable and enable build mode on hosts to update the live templates or choose to %s their configuration from 'Select Action' menu") % link_to_selected_hosts(_('recreate'), building_hosts(@template).map(&:id))).html_safe %>
```

Comment:
Probably obsolete given my comment above, but prefer .pluck(:id) rather than .map(&:id) as it'll run in SQL instead.